tests/Http/HttpRequestTest.php

```diff
@@ -646,7 +646,7 @@ public function testIntegerMethod()
         $this->assertSame(2, $request->integer('underscore_notation'));
         $this->assertSame(123456, $request->integer('unknown_key', 123456));
         $this->assertSame(0, $request->integer('null'));
-        $this->assertSame(0, $request->integer('null', 123456));
```

Comment:
This test is the proof that `get` and `input` were wrong. Why return `0` if the default was `123456`?